pkgs/applications/misc/ArchiSteamFarm/web-ui/default.nix

```
@@ -15,20 +13,13 @@ let
     sha256 = "1ajmi2l6xhv3nlnag2kmkblny925irp4gngdc3mniiimw364p826";
   };
 
-in
-nodePackages.package.override {
-  inherit src;
-
-  # upstream isn't tagged, but we are using the latest official commit for that specific asf version (assuming both get updated at the same time)
-  version = ArchiSteamFarm.version;
+  npmDepsHash = "sha256-AY1DFuZkB8tOQd2FzHuNZ31rtLlWujP+3AqsMMB2BhU=";
 
-  nativeBuildInputs = [ pkgs.nodePackages.node-gyp-build ];
+  nativeBuildInputs = [ nodePackages.node-gyp-build nodePackages.webpack-cli ];
 
-  postInstall = ''
-    patchShebangs node_modules/
-    npm run build
-    cp -r $out/lib/node_modules/asf-ui/dist $out/lib/dist
-    rm -rf $out/lib/node_modules/
+  installPhase = ''
+    mkdir $out
+    cp -rv dist/* $out/
```

```suggestion
    runHook preInstall
    mkdir $out
    cp -rv dist/* $out/
    runHook preInstall
```

Always run pre and post hooks when overriding the phase.